<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wk6440Calibration</class>
 <widget class="QDialog" name="wk6440Calibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>仪表校准</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	
	font: 11pt &quot;楷体&quot;;
	background-color: rgb(148, 148, 248);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>开短路校准</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btnOpen">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>36</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>开路
校准</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblOpen">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>工厂默认值。</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btnShort">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>36</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>短路
校准</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblShort">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>工厂默认值。</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>高频校准</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="btnHF">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>36</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>高频
校准</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblHF">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>工厂默认值。</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QDial" name="dial">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDone">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>36</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>完成</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>8</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
